core/modules/menu/menu.module

```diff
@@ -491,6 +491,12 @@ function menu_reset_item($link) {
 
 /**
  * Implements hook_block_info().
+ *
+ * This returns the blocks for any custom menus created on the site (as opposed
```

Comment:
This is why the `Menu` module was renamed `Menu UI` in Drupal 8. It  doesn't provide the ability to have menus, only the UI to create them :)

Reply:
Yes, I have now realized this 👍🏼